Sharprompt/Prompt.cs

```diff
@@ -24,5 +24,9 @@ public static T Select<T>(string message, IEnumerable<T> items, object defaultVa
         {
             return new Select<T>(message, items, defaultValue, pageSize, valueSelector ?? (x => x.ToString())).Start();
         }
+        public static IEnumerable<T> ComboBox<T>(string message, IEnumerable<T> items, object defaultValue = null, int pageSize = 10, int limit = -1, int min = 1, Func<T, string> valueSelector = null)
+        {
+            return new ComboBox<T>(message, items, limit, min, pageSize, valueSelector ?? (x => x.ToString())).Start();
+        }
```

Comment:
In general, ComboBox is almost the same as DropDownList and does not represent a multi-selectable GUI. Change to MultiSelect.